errors/errors.go

```diff
@@ -158,6 +158,7 @@ func FromError(err error) *StatusError {
 	if s, ok := err.(*ClientError); ok {
 		return &StatusError{grpcStatus: status.New(s.code, s.msg), origin: s.cause}
 	}
+	//nolint:errorlint // TODO: Should be fixed, but also heavily tested
```

Comment:
This shouldn't be here. Linter complains. If it doesn't complain locally, you have old version of GolangCi linter